src/locales/cn/translation.json

```diff
@@ -56,7 +56,9 @@
   },
   "purchase": {
     "donation": "捐款 - {{seller}}",
-    "voucher": "购买礼品券 - {{seller}}"
+    "voucher": "购买礼品券 - {{seller}}",
+    "donation_to": "Donation to {{seller}}",
```

Comment:
nit: I think we usually use camelCase for these

Reply:
yeah; i think you're right. but this is already used in existing places.  it would be too out of scope to adjust this. we can (and should) make a ticket to standardize the translations to bring it to parity.